### PR TITLE
[ca] Updating translations and exclude 

### PR DESCRIPTION
**Pull request by dansysanalyst**
Hello Andrey,

This PR updates some keys for Catalan language and adds an `excludes/ca.php` file with words matching English languages.

Some countries were updated according to the [Softcatala](https://www.softcatala.org/estandard-iso-catala/paisos/) ISO list.

This is my first contribution, please let me know if any review is needed.

Thank you

Dan

### CODE REVIEW

File: locales/ca/ca.json [hunk 1]
```diff
@@ -214,23 +214,23 @@
...
     "Extra confirmation is needed to process your payment. Please confirm your payment by filling out your payment details below.": "Extra confirmació és necessària per tramitar el seu pagament. Si us plau, confirmeu la vostra pagament per omplir les vostres dades de pagament a continuació.",
     "Extra confirmation is needed to process your payment. Please continue to the payment page by clicking on the button below.": "Extra confirmació és necessària per tramitar el seu pagament. Si us plau, continua a la pàgina de pagament per clic en el botó de sota.",
-    "Failed to load :resource!": "Failed to load :resource!",
+    "Failed to load :resource!": "Ha fallat en carregar :resource!",
     "Falkland Islands (Malvinas)": "Les Illes Malvines (Malvines)",
     "Faroe Islands": "Illes Fèroe",
     "February": "Febrer",
     "Fiji": "Fiji",
-    "Finish enabling two factor authentication.": "Finish enabling two factor authentication.",
+    "Finish enabling two factor authentication.": "Acabeu d'habilitar la autenticació de dos factors",
```
Comment (andrey-helldar) [inline locales/ca/ca.json:226]:
```suggestion
    "Finish enabling two factor authentication.": "Acabeu d'habilitar la autenticació de dos factors.",
```

File: locales/ca/ca.json [hunk 2]
```diff
@@ -606,30 +606,31 @@
...
+    "There are no fields to display.": "No hi ha camps per mostrar.",
+    "There are no new notifications": "No hi ha noves notificacions",
     "There is no active subscription.": "No hi ha una subscripció activa.",
     "There was a problem executing the action.": "Hi havia un problema d'executar l'acció.",
     "There was a problem submitting the form.": "Hi havia un problema d'enviar el formulari.",
     "These people have been invited to your team and have been sent an invitation email. They may join the team by accepting the email invitation.": "Aquestes persones han estat convidats al seu equip i han enviat una invitació per correu electrònic. Que poden entrar a l'equip per acceptar la invitació per correu electrònic.",
     "This account does not have an active subscription.": "Aquest compte no té una subscripció activa.",
-    "This copy of Nova is unlicensed.": "This copy of Nova is unlicensed.",
-    "This coupon code can only be used by new customers.": "This coupon code can only be used by new customers.",
+    "This action is unauthorized.": "Aquesta acció no està autoritzada.",
```
Comment (andrey-helldar) [inline locales/ca/ca.json:631]: This key is no longer used in Laravel products.

**Review by andrey-helldar — CHANGES_REQUESTED** [review]

File: locales/ca/ca.json (hunk 2 above)
Comment (andrey-helldar) [reply locales/ca/ca.json:631]:
```suggestion
```

**Comment by andrey-helldar** [comment]
Gràcies

**Review by andrey-helldar — APPROVED** [review]

**Comment by andrey-helldar** [comment]
One clarification: this is not my project. We have a team.

**Comment by dansysanalyst** [comment]
Spacibo! I will be sending more keys soon...

@andrey-helldar  I saw you were active on the last PRs and decided to address you. I apologize if this makes the team uncomfortable. Great project!

**Comment by andrey-helldar** [comment]
Do not worry. Everything is fine.

**Comment by andrey-helldar** [comment]
You can also use the localization installer: https://publisher.laravel-lang.com/installation/

**Pull request merged**